communities.md

```diff
@@ -281,6 +281,19 @@ The supported parameters are:
 
 It returns the top level communities in the repository (i.e. the communities that doesn't have a parent)
 
+#### findAdminAuthorized
+**/api/core/communities/search/findAdminAuthorized**
+
```

Comment:
Same here, a description would be good to add.  Something like:

`Returns all Communities in site for which the currently authenticated user has Admin privileges. No Communities are returned if the user does not have Admin privileges.`